lib/task-data/tasks/install-suse.js

```diff
@@ -11,12 +11,35 @@ module.exports = {
         osType: 'linux', //readonly options, should avoid change it
         profile: 'install-suse.ipxe',
         installScript: 'suse-autoinst.xml',
-        installScriptUri: '{{ api.templates }}/{{ options.installScript }}?nodeId={{ task.nodeId }}',
+        installScriptUri: '{{ api.templates }}/{{ options.installScript }}?nodeId={{ task.nodeId }}', //jshint ignore: line
         hostname: 'localhost',
         comport: 'ttyS0',
         repo: '{{file.server}}/distribution/{{options.version}}/repo/oss/',
         rootPassword: "RackHDRocks!",
-        kargs:{}
+        kargs:{},
+
+        progressMilestones: {
+            //jshint ignore: start
+            requestProfile:     { value: 1, description: 'Enter ipxe and request OS installation profile' },
+            enterProfile:       { value: 2, description: 'Enter profile, start to download installer'},
+            startInstaller:     { value: 3, description: 'Start installer and prepare installation' },
+            preConfig:          { value: 4, description: 'Enter Pre OS configuration'},
+            postPartitioning:   { value: 5, description: 'Finished partitioning and mounting, start package installation'},
+            chroot:             { value: 6, description: 'Finished package installation, start first boot'},
```

Comment:
This milestone definition refer to the SUSE auto-install instruction, this is the link http://users.suse.com/~ug/autoyast_doc/configuration.html#createprofile.scripts , I will add this link in code to avoid surprise.